package.json

```
@@ -41,13 +41,13 @@
     "react-player": "^2.6.2",
     "react-textarea-autosize": "^8.2.0",
     "seamless-immutable": "^7.1.4",
-    "stream-chat": "^2.2.2",
     "textarea-caret": "^3.1.0",
     "uuid": "^7.0.3"
   },
   "peerDependencies": {
     "react": "^16.8.x",
-    "react-dom": "^16.8.x"
+    "react-dom": "^16.8.x",
+    "stream-chat": "^2.2.x"
```

```suggestion
    "stream-chat": "2.2.x"
```
I'd suggest we lock in the version to only patches